common/src/main/java/com/duckduckgo/app/global/UriExtension.kt

```diff
@@ -180,3 +181,18 @@ fun Uri.getEncodedQueryParameters(key: String?): List<String> {
     } while (true)
     return Collections.unmodifiableList(values)
 }
+
+fun String.extractSchemeAndDomain(): String? {
```

Comment:
We can add some tests for this 2 new functions.

Reply:
Deferring, tracked in https://app.asana.com/0/0/1202592915624853/f